test/utils/snap.go

```diff
@@ -80,12 +81,13 @@ func SnapDumpLogs(t *testing.T, name string) {
 	fmt.Printf("Wrote snap logs to %s/%s\n", wd, filename)
 }
 
-// func SnapLogsJournal(t *testing.T, start time.Time, name string) {
-// 	Exec(t, fmt.Sprintf(
-// 		"journalctl --since \"%s\" --no-pager | grep \"%s\"\n\n",
-// 		time.Now().Format("2006-01-02 15:04:05"),
-// 		name))
-// }
+func SnapLogsJournal(t *testing.T, start time.Time, name string) string {
+	logs, _ := Exec(t, fmt.Sprintf(
+		"journalctl --since \"%s\" --no-pager | grep \"%s\"\n\n",
```

Comment:
When I wrote this function, it wasn't fetching the logs as expected. Maybe because it is missing the sudo rights.

Reply:
You are right, it needs the sudo rights.